edit-post/api/index.js

```diff
@@ -1,3 +1,8 @@
+/* eslint camelcase: 0 */
```

Comment:
This is going to be a hassle to maintain and scale, and if it's not easy to exempt might be worth considering a rename for experimental values to fall in-line with the rule.

Reply:
I changed it back to `_experimentalRegisterMoreMenuItem`